.github/workflows/test-pull_request_review.yml

```
@@ -0,0 +1,36 @@
+      - edited
+      - dismissed
+
+jobs:
+  approved:
+    runs-on: ubuntu-latest
+    steps:
+      - name: print
+        run: echo "${{ toJSON(github.event.review) }}"
+
+      - name: Add label to PR
+        uses: actions/github-script@v5
```

actionlint (1.7.4)

the runner of "actions/github-script@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)